compose.yml

```diff
@@ -6,30 +6,36 @@ services:
       - postgres_data:/var/lib/postgresql/data
       - ./db/postgres:/docker-entrypoint-initdb.d
     environment:
-      POSTGRES_HOST_AUTH_METHOD: trust
+      POSTGRES_USER: ats
+      POSTGRES_PASSWORD: password
+      POSTGRES_DB: ats_docker_production
     ports:
       - "5432:5432"
 
   web:
     build:
       context: .
-      dockerfile: development.Dockerfile
+      dockerfile: Dockerfile
       args:
-        - NODE_ENV=development
+        NODE_ENV: production
+        RAILS_ENV: docker_production
     command: bash -c "rm -f tmp/pids/server.pid && bundle exec rails s -b '0.0.0.0'"
     volumes:
       - .:/rails
+      - /rails/public/assets # To prevent deleting precompiled assets.
       - bundle:/usr/local/bundle
     ports:
       - "3000:3000"
     depends_on:
       - db
     environment:
-      - RAILS_ENV=development
-      - DATABASE_HOST=db
-      - DATABASE_USERNAME=ats
-      - BINDING=0.0.0.0
-      - PORT=3000
+      SECRET_KEY_BASE: 462d54c7b15dcc1924e797d0b28e6d
+      RAILS_ENV: docker_production
```

Comment:
Why do we need `docker_production` env?

Reply:
to use custom database configuration and to turn off SSL requirement

Reply:
will remove docker_production